sqlglot/executor/python.py

```diff
@@ -432,6 +432,7 @@ class Generator(generator.Generator):
             exp.Between: _rename,
             exp.Boolean: lambda self, e: "True" if e.this else "False",
             exp.Cast: lambda self, e: f"CAST({self.sql(e.this)}, exp.DataType.Type.{e.args['to']})",
+            exp.TryCast: lambda self, e: f"TRY_CAST({self.sql(e.this)}, exp.DataType.Type.{e.args['to']})" if e.args.get("safe") else f"CAST({self.sql(e.this)}, exp.DataType.Type.{e.args['to']})",
```

Comment:
I'd not deal with `safe` at all and just implement the "try cast or fallback" semantics.